extensions-core/parquet-extensions/src/test/java/org/apache/druid/data/input/parquet/CompatParquetReaderTest.java

```diff
@@ -88,22 +89,21 @@ public void testBinaryAsString() throws IOException
         JSONPathSpec.DEFAULT,
         false
     );
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Hi @cryptoe we have merged with changes from https://github.com/apache/druid/pull/15342 and used a common mapper object within the base class of these 2 classes. We shall squash the commits before merge. Please let us know if it seems fine otherwise